packages/jaeger-ui/src/components/Monitor/ServicesView/index.tsx

```diff
@@ -439,10 +439,10 @@ export function mapStateToProps(state: ReduxState): TReduxProps {
 }
 
 export function mapDispatchToProps(dispatch: Dispatch<ReduxState>): TDispatchProps {
-  const { fetchServices, fetchAllServiceMetrics, fetchAggregatedServiceMetrics } = bindActionCreators<any>(
-    jaegerApiActions,
-    dispatch
-  );
+  const { fetchServices, fetchAllServiceMetrics, fetchAggregatedServiceMetrics } = bindActionCreators<
+    {},
```

Comment:
`bindActionCreators` now requires a [mandatory `this` argument to be passed. ](https://github.com/reduxjs/redux/pull/2641/files#diff-042b5994aa33b68be7867734c0af5a6d1048e4a9722fb4db45ed7300689eb3d6R2)Previously, it was auto-assigned to [undefined.](https://github.com/reduxjs/redux/issues/2638)